app/models/concerns/hyrax/basic_metadata.rb

```diff
@@ -82,12 +82,16 @@ module BasicMetadata
 
       property :dateCreatedDisplay, predicate: ::RDF::Vocab::MODS.dateCreated  
 
-      property :dateCreatedIngest, predicate: ::RDF::Vocab::DC.created  
+      property :dateCreatedIngest, predicate: ::RDF::Vocab::DC.created
 
       property :label, predicate: ActiveFedora::RDF::Fcrepo::Model.downloadFilename, multiple: false
+
       property :relative_path, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#relativePath'), multiple: false
+
       property :import_url, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#importUrl'), multiple: false
 
+      property :descriptionNeighborhood, predicate: ::RDF::URI.new('http://scholarsphere.psu.edu/ns#descriptionNeighborhood'), multiple: false
```

Comment:
@summer-cook I believe `descriptionNeighborhood` can have multiple values, can we remove the `multiple: false`?